Makefile

```diff
@@ -138,4 +138,4 @@ benchmarks-comp:
 
 benchmarks-stargz:
 	@echo "$@"
-	@cd benchmark/stargzTest ; GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/StargzTests . && sudo ../bin/StargzTests $(COMMIT) ../singleImage.csv 10
+	@cd benchmark/stargzTest ; GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/StargzTests . && sudo ../bin/StargzTests $(COMMIT) ../singleImage.csv 10 $(stargzBinary)
```

Comment:
Can we default the variable to the default install location? (`/usr/local/bin/containerd-stargz-grpc`)

Reply:
Where does that "default" come from? I don't have it. The field is to pass in the ``containerd-stargz-grpc`` binary built by the stargz package.

Reply:
The default is the location that the stargz makefile will install the stargz snapshotter. It would be a nice convenience feature to be able to just run `make benchmarks-stargz` and have it run against the system installed stargz snapshotter. 

If we set the variable at the start of the makefile like this, we should be able to get both that default and the ability to pass it explicitly:
```
stargzBinary ?= /usr/local/bin/containerd-stargz-grpc
```

Side nit:
Is it idiomatic Make to name the variable with screaming snake case? (i.e. STARGZ_BINARY)

Reply:
Requested change is added in the latest revision.